content/post/data-warehouse-sla-p2.md

```diff
@@ -2,22 +2,23 @@
+In my previous post I explored the concept of a data warehouse SLA as a tool to build a trusting stakeholder relationship. I also introduced a template to help you write your very own data warehouse SLA - a letter addressed to your stakeholder. In this post I walk through the meat of the SLA template: services provided, expected performance, problem reporting, response time, monitoring processes, issue communication and stakeholder commitment. If you have not read [part one]({{< relref "data-warehouse-sla-p1.md" >}}), I highly recommend reading it first!
+<!--more-->
 
 ## Writing you SLA
 
 ### Services provided
-At a 30,000 foot level you are providing data access. However, I recommend going as specific as possible when describing the data services provided. Greater specificity helps to align expectations on which fields are included and the definition of those fields (specificity also helps to frame future discussions when adding additional fields to the data warehouse). Field definition becomes increasingly important when the data is used to calculate individual performance and compensation.
+At a 30,000 foot level the data warehouse provides data access. However, I recommend going as specific as possible when describing the data services provided in your SLA. Greater specificity helps to align expectations on which fields are included and the definition of those fields (specificity also helps to frame future discussions when adding additional fields to the data warehouse). Field definition becomes increasingly important when the data is used to calculate individual performance and compensation (e.g., sales team commission, executive bonuses).
 
-The counter point is that you don't want to go overboard - you are not providing value to the business if you spend all your time working on a data dictionary. You should not expect to have a detailed data dictionary early in the life of your data warehouse. Find the right balance between specificity and the resources available to you.
+The counter point is that you don't want to go overboard - you are a data team, not a documentation team. You are not discovering new insights and unlocking value for the business if you spend all your time working on a data dictionary. The perfect level of data, unfortunately, is a moving target and it will continue to change as your business grows in size and complexity. To be clear, you should not expect to have a detailed data dictionary when you are a ten person startup. Find the right balance between specificity and the resources available to you.
 
 Here are four examples of services provided with increasing specificity:
```

Comment:
Is this a cascading list? i.e., Don't work on #4 if you haven't already done #1 - 3